web/html/src/components/input/FormMultiInput.js

```diff
@@ -20,3 +36,70 @@ export function getOrderedItemsFromModel(model: Object, prefix: string): Array<s
       return num1 - num2;
     });
 }
+
+/**
+ * Component handling list of items in forms
+ */
+export function FormMultiInput (props: Props) {
+  const formContext = React.useContext(FormContext);
+  const items = getOrderedItemsFromModel(formContext.model, props.prefix);
+  const new_index = (items.length > 0 && Number.parseInt(items[items.length - 1].substring(props.prefix.length), 10) + 1) || 0;
```

Comment:
Following the suggestion to make `getOrderedItemsFromModel` return an array of indices instead;
This calculation can be a lot simpler:
```js
const indices = getOrderedIndicesFromModel(formContext.model, props.prefix);
const new_index = Math.max(...indices) + 1 || 0;
```
Also you'd have indices instead in the loop below, so you wouldn't need to parse it again.

Reply:
done